packages/babel-jest/src/__tests__/index.ts

```diff
@@ -5,12 +5,15 @@
  * LICENSE file in the root directory of this source tree.
  */
 
+import type {TransformOptions as BabelTransformOptions} from '@babel/core';
+import type {EncodedSourceMap} from '@jridgewell/trace-mapping';
```

Comment:
Or would it be better to reexport from `jest/transform`? Not sure..

Reply:
Yes